python/private/py_runtime_rule.bzl

```diff
@@ -388,11 +381,7 @@ The {obj}`PyRuntimeInfo.zip_main_template` field.
 )
 
 def _is_singleton_depset(files):
-    # Bazel 6 doesn't have this helper to optimize detecting singleton depsets.
-    if _py_builtins:
-        return _py_builtins.is_singleton_depset(files)
```

Comment:
Please keep the py_builtins.is_singleton_depset. It's a minor optimization, but I like it.